rgw/v2/tests/aws/test_sts_rename_large_object.py

```diff
@@ -111,6 +111,12 @@ def test_exec(config, ssh_con):
         )
     )
     utils.exec_shell_cmd(add_caps_cmd)
+    if admin_user:
```

Comment:
From where do we get value for admin_user?

Reply:
Removed.